app/lib/efile/az/az140.rb

```diff
@@ -59,6 +59,7 @@ def calculate
           set_line(:AZ140_LINE_80, :calculate_line_80)
         end
         set_line(:AZ140_LINE_79, :calculate_line_79)
+        set_line(:AZ140_LINE_97, @intake, :prior_last_names)
```

Comment:
my conception of this layer (the so-called "calculator classes") is that we only need to put numbers and booleans in them, things like names and birthdates only need to be added to the xml/pdf layers as they're needed

Reply:
gotcha, removing